communication/data_logging.cpp

```diff
@@ -42,15 +42,15 @@
 
 #include "communication/data_logging.hpp"
 
+#include <cstdlib>
+#include <math.h>
```

Comment:
No real reason, fixed


On 25/07/16 08:38, GregoireH wrote:
>
> In communication/data_logging.cpp 
> <https://github.com/lis-epfl/MAVRIC_Library/pull/278#discussion_r72014626>:
>
> > @@ -42,15 +42,15 @@
> >
> >  #include "communication/data_logging.hpp"
> >
> > +#include <cstdlib>
> > +#include <math.h>
>
> Could you clarify if you forget to put math.h back within extern C or 
> if for some reason it can remain up there please ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/lis-epfl/MAVRIC_Library/pull/278/files/50dd3454a89564410f321f0be6e9a99188fe21ff#r72014626>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABUMh1IiYI1LX7_qA8GU3DQCKq7TjNo0ks5qZFnxgaJpZM4JOlA9>.
>